mistralrs-core/src/pipeline/mod.rs

```diff
@@ -18,7 +18,7 @@ mod sampling_pipeline;
 use crate::lora::{LoraConfig, Ordering};
 use crate::DeviceMapMetadata;
 use candle_core::quantized::GgmlDType;
-use chat_template::ChatTemplate;
+pub use chat_template::ChatTemplate;
```

Comment:
Not sure if this should be public, but seems like might have been required for the docs reference to it. The struct itself is annotated as `pub` and not `pub(crate)` so I think that's ok?